core/common/src/ChangedElements.ts

```diff
@@ -11,26 +11,57 @@ import { AxisAlignedBox3dProps } from "./geometry/Placement";
  * @module Entities
  */
 
-/** @internal */
+/** Type of change bitflags that show what changed on an element
+ * @public
+ */
 export enum TypeOfChange {
+  /** A property in the element changed */
   Property = 0b1,
+  /** The geometry stream of a geometric element changed */
   Geometry = 0b10,
+  /** The placement of a geometric element changed */
   Placement = 0b100,
+  /** Indirect change occurred to this element by a related instance */
   Indirect = 0b1000,
+  /** Hidden properties of the element changed */
   Hidden = 0b10000,
 }
 
-/** @internal */
+/** Changed elements found in a changeset or between a range of changesets
```

Comment:
Does `classIds` have the same number of entries as `elements` such that the classId of elements[x] is always classIds[x]?
Is the same true of `modelIds` and `types`?

Reply:
Yes, all arrays will have the same length, and an index relates to a single element.

Reply:
Please make the docs say that.

Reply:
Pushed